scipy/optimize/_minimize.py

```diff
@@ -65,6 +65,11 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
         where ``x`` is a 1-D array with shape (n,) and ``args``
         is a tuple of the fixed parameters needed to completely
         specify the function.
+
+        Suppose the callable has signature ``f0(x, *args, **kwargs)``, where ``args``
```

Comment:
If you'd prefer, this could be phrased "Suppose the callable requires positional arguments besides `x` and additional keyword arguments, and these are collected into a tuple `args` and dictionary `kwargs`, respectively." But I prefer what is here to that.

Reply:
If you are split and need a casting vote, I would vote for the rephrasing.

Reply:
I'll try to clarify my original concern, in case it is helpful: the user is reading the function signature + docstring while getting this message, so it is quite natural to think of `*args` and `**kwargs` to pertain to the current function. Related, a common pattern (perhaps not in SciPy, but in the ecosystem) is to allow such args and kwargs as additional arguments as, e.g., `callable_args` and `callable_kwargs`. I think the approach suggested here is a perfectly fine—a better alternative, even—as long as it is made clear to the user that *they* need to gather the args, wrap a function around it, and send that function into SciPy.

Obviously, I understand what to do myself, but when reviewing / writing docstrings, I try to put on a hat of someone who is competent with Python but not familiar with unusual (or project-specific) idioms.

Reply:
Please make an inline suggestion, such as the alternative I proposed if you prefer that. Once that is concrete, I can consider committing it. 

I understand putting on the hat of the reader... I just have a different opinion in this case.

Reply:
I see now that we complicate things further by allowing `args`. I see three viable paths:

(1) Continue allowing args to be specified, and then say nothing more about keyword arguments, or callable wrapping.
(2) Extend `args` to have a `kwargs` dual, and change the signature to: `            fun(x, *args, **kwargs) -> float.
(3) Deprecate `args`, and describe the callable wrapping workaround in the docs.

I don't see how mixing callable wrapping and args specification can ever be anything but confusing. Easiest for now is to do (1), i.e. to simply remove the wrapping text.

Reply:
The wrapping text is the purpose of the PR. Please see the linked issue.